yesod-core/Yesod/Core/Class/Yesod.hs

```diff
@@ -366,6 +366,22 @@ sslOnlySessions = (fmap . fmap) secureSessionCookies
     setSecureBit cookie = cookie { setCookieSecure = True }
     secureSessionCookies = customizeSessionCookies setSecureBit
 
+-- | Helps defend against CSRF attacks by setting the SameSite attribute on
+-- session cookies to "Lax".
```

Comment:
thought about this, and I think we should have:

* Link to https://datatracker.ietf.org/doc/draft-west-first-party-cookies/
* Mention that the cookie option is new and changeable.
* Explain briefly what the (current) difference is between Lax and Strict.